bciers/apps/reporting/src/app/components/taskList/taskListPages/5_signOffSubmit.ts

```diff
@@ -40,64 +39,6 @@ export const signOffSubmitPageFactories: {
       link: `/reports/${reportVersionId}/sign-off`,
       isActive: activePage === ReportingPage.SignOff,
     },
+    continueUrl: `/reports/${reportVersionId}/submission`,
   }),
 };
-
-export enum ActivePage {
-  "Verification" = 0,
-  "Attachments",
-  "FinalReview",
-  "SignOff",
-}
-export const getSignOffAndSubmitSteps: (
-  versionId: number,
-  activeIndex?: ActivePage | number,
-  needsVerification?: boolean,
-) => Promise<TaskListElement[]> = async (
-  versionId,
-  activeIndex = undefined,
-  needsVerification = false,
-) => {
-  // Build the TaskListElement array
-  const elements: TaskListElement[] = [
-    {
-      type: "Page", // Set the type to "Page"
-      title: "Final review",
-      link: `/reports/${versionId}/final-review`,
-      isActive: activeIndex === ActivePage.FinalReview,
-    },
-    {
-      type: "Page", // Set the type to "Page"
-      title: "Verification",
-      link: `/reports/${versionId}/verification`,
-      isActive: activeIndex === ActivePage.Verification,
-    },
-    {
-      type: "Page", // Set the type to "Page"
-      title: "Attachments",
-      link: `/reports/${versionId}/attachments`,
-      isActive: activeIndex === ActivePage.Attachments,
-    },
-    {
-      type: "Page", // Set the type to "Page"
-      title: "Sign-off",
-      link: `/reports/${versionId}/sign-off`,
-      isActive: activeIndex === ActivePage.SignOff,
-    },
-  ];
-  // Conditionally filter out "Verification" based on needsVerification is false
-  const filteredElements: TaskListElement[] = elements.filter((element) => {
-    if (!needsVerification) {
-      return element.title !== "Verification";
-    }
-    return true; // If needsVerification is true, keep all elements
-  });
-  return [
-    {
-      type: "Section",
-      title: "Sign-off & submit",
-      isExpanded: true,
-      elements: filteredElements,
-    },
-  ];
-};
```

Comment:
Pretty sure this got removed in a separate PR...something might be wonky with rebases?